bouncer/src/repo/manipulator/modelconvertor/import/odaHelper/data_collector_oda.h

```diff
@@ -25,56 +25,51 @@
 #include <string>
 #include <fstream>
 
-#include "vectorise_device.h"
+#include "vectorise_device_dgn.h"
 #include "../../../../core/model/bson/repo_node_mesh.h"
 
 #include <vector>
+#include "material_colors.h"
+
 namespace repo {
 	namespace manipulator {
 		namespace modelconvertor {
 			namespace odaHelper {
-				class GeometryDumper : public OdGiGeometrySimplifier, public OdGsBaseMaterialView
+
+				class DataCollectorOda : public OdGiGeometrySimplifier, public OdGsBaseMaterialView
```

Comment:
Call this DataProcesser. 

You're already in odaHelper there's not a lot opoint appending Oda at the end of it.
This class is more of a processor than collecting anything (GeometryCollector is the one that's Collecting something).